packages/front/src/App.tsx

```diff
@@ -54,15 +55,62 @@ export default function App() {
   useVerifyUserLogin({ user, setUser }, navigate);
 
   return (
-    <ErrorBoundary
-      FallbackComponent={({ error, resetErrorBoundary }) => {
-        resetErrorBoundary();
-        return <></>;
-      }}
-    >
-      <Suspense fallback={<div>Loading...</div>}>
-        <Outlet context={{ user, setUser }} />
-      </Suspense>
-    </ErrorBoundary>
+    <>
+      <FlexComponent
+        flexProps={{ container: true, col: true }}
```

Comment:
i have mixed feeling about these props. but if you like keep it

Reply:
might something more high level like variants are better. you can also compose variants. check https://github.com/joe-bell/cva